spec/datatables/supervisor_datatable_spec.rb

```diff
@@ -0,0 +1,59 @@
+require "rails_helper"
+
+RSpec.describe "SupervisorDatatable" do
+  subject { described_class.new(org.supervisors, params).as_json }
+
+  let(:org) { create(:casa_org) }
+  let(:order_by) { "display_name" }
+  let(:order_direction) { "asc" }
+  let(:params) { datatable_params(order_by: nil, additional_filters: additional_filters) }
+
+  describe "filter" do
+    let!(:active_supervisor) { create(:supervisor, casa_org: org, active: true) }
+    let!(:inactive_supervisor) { create(:supervisor, casa_org: org, active: false) }
+
+    describe "active" do
+      context "when active" do
+        let(:additional_filters) { {active: %w[true]} }
+
+        it "brings only active supervisors", :aggregate_failures do
+          expect(subject[:recordsTotal]).to eq(2)
```

Comment:
as all tests are the same, could you use shared example? and could you change this tests using map? I think this is not a good approach